guide/src/whirlwind-tour/basic-usage.md

```diff
@@ -112,7 +114,8 @@ js.then(js => {
 
 Note that we're using `import(..)` here because Webpack [doesn't
 support][webpack-issue] synchronously importing modules from the main chunk just
-yet.
+yet. The `file-loader` plugin will copy your `index.html` file when you publish
```

Comment:
Instead of saying, it will publish the application. It would be great to explain it here

Reply:
I'm not sure what you mean. Is "publishing" a poor term for "getting something that can run outside of the dev server"? If so, can you suggest a different one?

Reply:
wasm-pack too publishes your application and definitely you are not talking about the same publication here. Doesn't it make sense to explain that?  (the overloading of application is confusing)

Reply:
Done